src/xunit.netcore.extensions/xunit.netcore.extensions.csproj

```diff
@@ -10,6 +10,8 @@
     <RootNamespace>Xunit.NetCore.Extensions</RootNamespace>
     <CLSCompliant>false</CLSCompliant>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
```

Comment:
Just tried this. Buildtools compiled without the <TargetFrameWork*> tags causes the following error once it is imported into CoreFX and then CoreFX tests are ran:
```
CoreRun.exe xunit.console.netcore.exe System.Collections.Tests.dll  -xml testResults.xml -notrait category=nonwindowstests  -notrait ca
  tegory=OuterLoop -notrait category=failing

  Unhandled Exception: System.IO.FileNotFoundException: Could not load file or assembly 'xunit.runner.utility.dnx, Version=2.1.0.3029, Cu
  lture=neutral, PublicKeyToken=null' or one of its dependencies. The system cannot find the file specified.
     at Xunit.ConsoleClient.Program.Main(String[] args)
```